templates/master/00-master/baremetal/files/baremetal-keepalived-keepalived.yaml

```diff
@@ -10,7 +10,7 @@ contents:
     }
 
     vrrp_script chk_dns {
-        script "/usr/bin/host -t SRV _etcd-server-ssl._tcp.{{ .EtcdDiscoveryDomain }} localhost"
+        script "/usr/bin/host api-int.{{ .EtcdDiscoveryDomain }} localhost"
```

Comment:
I think we should stop using the etcd discovery domain all together...

Reply:
I couldn't find where to get the base domain from. Any suggestion?

Reply:
I think the baremetal-runtime-cfg has a way to provide the cluster domain at runtime.. that might be an option?

Reply:
I can change it here to `{{ .Cluster.Domain }}` but many of the templates are not rendered via baremetal-runtimecfg, so I'd rather leave `{{ .EtcdDiscoveryDomain }}` alone until we decide to effectively get rid of it globally.

Reply:
> I can change it here to {{ .Cluster.Domain }} but many of the templates are not rendered via baremetal-runtimecfg, so I'd rather leave {{ .EtcdDiscoveryDomain }} alone until we decide to effectively get rid of it globally.

api-int.<etcd discovery domain> is plain wrong imo, it works because that's the equal today. so i don't see why we would continue to use it..?

Reply:
OK, I misunderstood your first comment - I'll make the change.

Reply:
This is just checking that the DNS server is responding. It doesn't matter what record we look up. api-int happens to be a known name that should always be there.

Reply:
> This is just checking that the DNS server is responding. It doesn't matter what record we look up. api-int happens to be a known name that should always be there.

api-int.<cluster domain> is a known DNS name that should always be there, and not api-int.<etcd discovery domain>

Reply:
Ah, Github ate the domain part of your comment. I thought you were objecting to the use of api-int in general. :-)